docs/user-roles.rst

```diff
@@ -0,0 +1,39 @@
+.. _user-roles:
+
+==========
+User Roles
+==========
+
+An overview of user roles and their privileges for organizations and projects.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. _org-roles:
+
+Organization roles
+==================
+
+An *Organization Admin* can add users to and remove users from an organization.
+Admins can perform all available operations for projects and services. They have access to the organization's Audit Log.
+Each organization must have at least one Admin.
+
+An *Organization Member* is able to view the list of organization users but can't edit, add, or remove users.
+They only have access to projects they are part of.
+
+.. _project-roles:
+
+Project roles
+=============
+
+A Project Admin can add users to and remove users from a project. They can also perform every available operation
```

Comment:
I am wondering: do we really want to capitalize noun phrases like "Project Admin". it makes the docs sound a little stuffy from my pov. I get that we're talking about like Official Concepts, but I think it's unnecessary here (and maybe a little too German haha)

Reply:
I also debated this a bit. Initially it was done inconsistently, but I decided on capitalizing throughout because it might make it clearer that these are technical terms - specific roles to which particular privileges attach - rather than just descriptive terms. But I'm not super wedded to it either, so I'm happy to lowercase them.

Reply:
it's okay, let's leave them for now